src/deluge/gui/fonts/fonts.c

```diff
@@ -2681,7 +2681,7 @@ const uint8_t font_metric_bold_20px[] = {
         0b00001111, 0b00000000, 0b00000000,
         0b00001111, 0b00000000, 0b00000000,
         0b00001111, 0b00000000, 0b00000000,
-        0b00011111, 0b00000000, 0b00000000,
```

Comment:
what's this?

Reply:
Fixed an erroneous pixel on the T font

It looked like this

.......
 ....
  ...
  ...